package.json

```diff
@@ -50,5 +50,11 @@
   "engines": {
     "node": "^4.5 || 6.* || >= 7.*"
   },
-  "private": true
+  "private": true,
+  "dependencies": {
+    "architect-scripting": "git+ssh://git@bitbucket.org:inindca/architect-scripting.git",
```

Comment:
You can't have a private repo referenced in a public repo.  If someone w/o inindca bitbucket access wanted to run dev tools locally they wouldn't be able to.

Reply:
That makes sense. I will look into getting a public package of it setup.